codegen/src/main/java/software/amazon/awssdk/codegen/model/config/customization/CustomizationConfig.java

```diff
@@ -303,4 +304,12 @@ public String getSdkResponseBaseClassName() {
     public void setSdkResponseBaseClassName(String sdkResponseBaseClassName) {
         this.sdkResponseBaseClassName = sdkResponseBaseClassName;
     }
+
+    public String getDefaultExceptionUnmarshaller() {
```

Comment:
How easily can these be applied to json or rest-json services?

Reply:
JSON-based services don't support customizing the unmarshallers at all, so that would need to be added to their protocol spec (similar to how it is in XML).